recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -136,9 +156,19 @@ def package_info(self):
         postfix = "d" if self.settings.os == "Windows" and self.settings.build_type == "Debug" else ""
         # TODO: back to global scope in conan v2 once cmake_find_package* generators removed
         self.cpp_info.components[libname].libs = [f"{libname}{postfix}"]
-        self.cpp_info.components[libname].requires = ["zeromq::zeromq", "cppzmq::cppzmq", "ncurses::ncurses"]
-        if self.options.with_coroutines:
+        self.cpp_info.components[libname].requires = ["zeromq::zeromq"]
+        
+        if Version(self.version) < "3.6.0":
             self.cpp_info.components[libname].requires.append("boost::coroutine")
+
+        if Version(self.version) < "4.1.0":
+            self.cpp_info.components[libname].requires.extend(["ncurses::ncurses"])
+            if self.options.with_coroutines:
+                self.cpp_info.components[libname].requires.append("boost::coroutine")
+
+        if Version(self.version) > "4.1.0" and self.options.with_sqlite_logging:
```

Comment:
change to 4.1.1 ?

Reply:
I think SQLite is needed for 4.1.1 and more, right? (so > 4.1.0)

Reply:
right